dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/impl/AsyncResponseImpl.java

```diff
@@ -354,7 +354,11 @@ public synchronized void handleTimeout() {
     private void initContinuation() {
         ContinuationProvider provider =
             (ContinuationProvider)inMessage.get(ContinuationProvider.class.getName());
-        cont = provider.getContinuation();
+        if (provider != null) {
+            cont = provider.getContinuation();
+        } else {
+            throw new IllegalArgumentException("All servlets and servlet filters must support async!");
```

Comment:
The message text seems a little harsh.  Maybe something like:
```suggestion
            throw new IllegalArgumentException("Continuation not supported. Please ensure that all servlets and servlet filters must support async operations");
```